Packs/Base/Scripts/DBotTrainClustering/DBotTrainClustering.yml

```diff
@@ -0,0 +1,153 @@
+args:
+- default: false
+  description: Comma-separated list of incident fields to take into account when training
+    the clustering.
+  isArray: false
+  name: fieldsForClustering
+  required: false
+  secret: false
+- default: false
+  description: Incident field which represent the family name for each cluster created.
```

Comment:
Need to extend the explanation here, it's more than just the name, also helps to characterize each cluster

Reply:
ok

Reply:
agree